sdk/tables/data-tables/test/internal/isCosmosEndpoint.spec.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { describe, it, assert } from "vitest";
+import { isCosmosEndpoint } from "../../src/utils/isCosmosEndpoint.js";
+
+describe("isCosmosEndpoint", () => {
+  it("returns true for cosmosdb.* host", () => {
+    const result = isCosmosEndpoint("https://abc.table.cosmosdb.azure.com");
+    assert.equal(result, true);
```

Comment:
Nit, would we want to just use isTrue or isFalse?

Reply:
Of course!

Reply:
Updated